src/components/MissionProfiler.js

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import './missions.css';
+
+const MyProfile = () => {
+  const { missions } = useSelector((state) => state.missions);
+  const tourists = missions.filter((visit) => visit.mission);
+  return (
+    <div>
+      <h1 className="head-text">My Missions</h1>
+      <table className="tour_list">
+        <tbody className="tour_body">
+          {
+            tourists.length < 1 ? (
+              <tr>
+                <td className="tour-now tour">No Mission Yet...</td>
+                <td><Link to="/missions">Join a Mission</Link></td>
```

Comment:
- [ ] Good job 👏🏿 adding a placeholder message when the "My Missions" or "My Rockets" lists are empty